src/lib/utils/wrap.js

```diff
@@ -0,0 +1,11 @@
+export default function wrap(Type, fn, dontCloneWithNoArgs) {
```

Comment:
I would put exactly how this works in the contributing file. Also needs to be a standard part of code review to make sure this gets used if it needs to be.